data/templates/scripts/to_powershell.ducky_script.template

```diff
@@ -0,0 +1,15 @@
+REM Title: Metasploit Generated Payload
+REM Description: Opens a payload via powershell on the system
+REM Target: Windows 10/11 (Powershell)
```

Comment:
I tested this on Windows 7 SP1 and Server 2019, in both cases, it worked just fine. Do we need this detail here? It sort of makes it sound like this will only work on Windows 10 and Windows 11.

Reply:
not required, just saw others had it. prob better to remove it